pyseir/load_data.py

```diff
@@ -325,6 +331,9 @@ def load_new_case_data_by_fips(
     """
     timeseries = combined_datasets.build_us_timeseries_with_all_fields()
     county_case_data = timeseries.get_data(None, fips=fips)
+    # Only including rows with cases reported.
+    county_case_data = county_case_data.loc[county_case_data[CommonFields.CASES].notnull(), :]
```

Comment:
There was already a potential bug in that the returned arrays could be missing days in the middle of the range without a way to represent the gap. The filter added here makes that problem more likely to occur.

Reply:
A clean fix, that potentially is out of scope for this change is returning a Series with a `date` index.

Reply:
Yeah, I more and more don't like this.  I'm trying to think of the best way to handle missing data.  another options is to fill in the missing days more explicitly. 

the issue that this is indirectly trying to fix is: 
https://sentry.io/organizations/covidactnow/issues/1734408534/?project=5203044&query=is%3Aunresolved

I think a better solution might be to just leave the nans and add an epsilon to the denominator here: https://github.com/covid-projections/covid-data-model/blob/master/pyseir/inference/infer_rt.py#L740